stage1/init/common/pod.go

```diff
@@ -1046,3 +1052,26 @@ func parseLinuxCapabilitiesSet(capSet types.LinuxCapabilitiesSet) []string {
 	}
 	return capsStr
 }
+
+func getAppNoNewPrivileges(isolators types.Isolators) (bool, error) {
+	noNewPrivs := ""
+
+	// TODO(sur): once https://github.com/appc/spec/pull/611 lands, bump appc spec, and use official appc types instead
+	for _, isolator := range isolators {
+		if isolator.Name == "os/linux/no-new-privs" {
+			noNewPrivs = string(*isolator.ValueRaw)
+			break
```

Comment:
i.e. if there are multiple `"os/linux/no-new-privs"` instances in the manifest? If so, yes, that is a very good idea, the most restrictive one in the case of bool would be an or'ed version of all no-new-privs instances in the manifest.